src/backend/storeManagers/nile/user.ts

```diff
@@ -115,4 +115,8 @@ export class NileUser {
 
     return user.extensions.customer_info
   }
+
+  public static isLoggedIn() {
+    return configStore.get_nodefault('userData') || false
```

Comment:
please @imLinguin or @Gustash can you confirm if this is the correct way to check if we are logged in?

Reply:
configStore is populated once when we migrate an existing nile config, if it exists, and whenever a user logs in/out, so this should be valid